rerun_py/src/python_bridge.rs

```diff
@@ -1106,13 +1106,16 @@ fn get_app_url() -> String {
     }
 
     let build_info = re_build_info::build_info!();
-    if let Some(short_git_hash) = build_info.git_hash.get(..7) {
-        format!("https://rerun.io/viewer/commit/{short_git_hash}")
+
+    if build_info.is_final() {
+        format!("https://app.rerun.io/version/{}", build_info.version)
+    } else if let Some(short_git_hash) = build_info.git_hash.get(..7) {
+        format!("https://app.rerun.io/commit/{short_git_hash}")
     } else {
         re_log::warn_once!(
-            "No valid git hash found in build info. Defaulting to rerun.io/viewer for app url."
+            "No valid git hash found in build info. Defaulting to app.rerun.io for app url."
         );
-        "https://rerun.io/viewer".to_owned()
+        "https://app.rerun.io".to_owned()
```

Comment:
This should have a comment explaining _why_ this goes to `app.rerun.io` instead of `rerun.io/viewer`; otherwise someone may want to change it again :)